cluster/pulumi/infra/src/istio.ts

```diff
@@ -326,7 +378,8 @@ function configureGatewayService(
         },
         service: {
           loadBalancerIP: ingressIp,
-          loadBalancerSourceRanges: externalIPRanges,
+          // We limit IPs using istio instead of through loadBalancerSourceRanges as the latter has a size limit.
```

Comment:
Can we also add a link to the original issue with all the investigation just in case someone in a few years gets ideas about trying it again?

Reply:
added, it's a link on the internal repo but still seems better to have it than not to